src/libs/elektra/kdb.c

```diff
@@ -378,6 +378,7 @@ static KDB * kdbNew (Key * errorKey)
 	handle->global =
 		ksNew (1, keyNew ("system:/elektra/kdb", KEY_BINARY, KEY_SIZE, sizeof (handle), KEY_VALUE, &handle, KEY_END), KS_END);
 	handle->backends = ksNew (0, KS_END);
+	handle->hooks = elektraCalloc (sizeof (struct _Hooks));
```

Comment:
As stated above, I'd get rid of all the `calloc`s for the hooks, but this one is extra unnecessary. AFAICT this will never be used and will always be freed by `initHooks`.

Reply:
Ha that one is actually required currently! When bootstrapping KDB `initHooks` isnt called. 🤷‍♂️. But I will inline the struct, so it will no longer be necessary.